packages/world/src/utils/PlayerUtils.sol

```diff
@@ -56,3 +58,49 @@ function requireInPlayerInfluence(VoxelCoord memory playerCoord, EntityId entity
   requireInPlayerInfluence(playerCoord, coord);
   return coord;
 }
+
+function createPlayer(EntityId playerEntityId, VoxelCoord memory playerCoord) {
```

Comment:
I tried putting these two new utils in `EntityId` but causes a circular import since `VoxelCoord` already imports `EntityId`